datahub-graphql-api/README.md

```diff
@@ -0,0 +1,61 @@
+Notice: `datahub-graphql-api` is currently in beta, and as such is currently subject to backwards incompatible changes. 
+
+# DataHub GraphQL API
+DataHub GraphQL API is a shared lib module containing a GraphQL API on top of the GMS service layer. It exposes a graphical representation
+permitting reads and writes against the entities and aspects on the Metadata Graph, including Datasets, CorpUsers, & more. 
+
+Contained within this module are 
+
+1. **GMS Schema**: A GQL schema that based on GMS models, located under `resources/gms.graphql`. 
+2. **GMS Data Fetchers**: Components used by the GraphQL engine to resolve individual fields in the GQL schema.
+3. **GMS Data Loaders**: Components used by the GraphQL engine to fetch data from downstream sources efficiently (by batching). 
+4. **GraphQLEngine**: A wrapper on top of the default `GraphQL` object provided by `graphql-java`. Provides a way to configure all of the important stuff using a simple `Builder API`. 
+5. **GMSGraphQLEngine**: An engine capable of resolving the GMS schema using the data fetchers + loaders mentioned above (with no additional configuration required). 
+
+We've chosen to place these components in a library module so that GraphQL servers can be deployed in multiple "modes":
+
+1. **Standalone**: GraphQL facade, mainly used for programmatic access to the GMS graph from a non-Java environment
+2. **Embedded**: Currently, we are leveraging these common components to surface an extended GQL API from `datahub-frontend` 
```

Comment:
Suggested rewording:
**Embedded**: Leverage-able within another Java server to surface an extended GQL API. e.g. We use this to add GraphQL support to `datahub-frontend`